PWGDQ/Core/VarManager.h

```diff
@@ -2510,6 +2510,15 @@ void VarManager::FillDileptonHadron(T1 const& dilepton, T2 const& hadron, float*
   if (fgUsedVars[kDeltaEta]) {
     values[kDeltaEta] = dilepton.eta() - hadron.eta();
   }
+  if (fgUsedVars[kMass]) {
+    values[kMass] = dilepton.mass();
+  }
+  if (fgUsedVars[kPt1]) {
+    values[kPt1] = dilepton.pt();
+  }
+  if (fgUsedVars[kPt2]) {
+    values[kPt2] = hadron.pt();
+  }
```

Comment:
These variables are filled already in this function. They have different names, but exist just in the same function. Also, the kMass, kPt1 and kPt2 are used for very different purposes and there is a potential for confusion when filling histogram classes.